sdk/python/tests/example_feature_repo_1.py

```diff
@@ -65,3 +72,14 @@
     input=customer_driver_combined_source,
     tags={},
 )
+
+
+all_drivers_feature_service = FeatureService(
+    name="driver_locations_service",
+    features=[driver_locations],
+    tags={"release": "production"},
+)
+
+partial_drivers_feature_service = FeatureService(
```

Comment:
Is `partial_drivers_feature_service` being used anywhere?

Reply:
nope, deleted